web/templates/pages/members.html

```diff
@@ -10,39 +10,22 @@
-				<li><a href='https://www.facebook.com/asokica?fref=grp_mmbr_list'><img src="{% static "img/heidi.png" %}">Heidi Pungartnik</a></li>
-				<li><a href='https://www.facebook.com/petra.vidmar.14?fref=grp_mmbr_list'><img src="{% static "img/petra.png" %}">Petra Vidmar</a></li>
-				<li><a href='https://www.facebook.com/nina.ninchy2?fref=grp_mmbr_list'><img src="{% static "img/ninaninchy.png" %}">Nina Pristov</a></li>
-					<li><a href='https://www.facebook.com/mojca007?fref=grp_mmbr_list'><img src="{% static "img/mojca.png" %}">Mojca</a></li>
-				<li><a href='https://www.facebook.com/dobai.livia.5?fref=grp_mmbr_list'><img src="{% static "img/liviadobai.png" %}">Livia Dobai</a></li>
-				<li><a href='https://www.facebook.com/masa.c.logar?fref=grp_mmbr_list'><img src="{% static "img/masalogar.png" %}">Maša Černovšek Logar</a></li>
-				<li><a href='https://www.facebook.com/nina.malovrh.1?fref=grp_mmbr_list'><img src="{% static "img/ninamalovrh.png" %}">Nina Malovrh</a></li>
-				<li><a href='https://www.facebook.com/profile.php?id=1186838569&fref=grp_mmbr_list'><img src="{% static "img/hanaisnouk.png" %}">Hana Isnouk</a></li>
-				<li><a href='https://www.facebook.com/pijab1?fref=grp_mmbr_list'><img src="{% static "img/pijabalaban.png" %}">Pija Balaban</a></li>
-				<li><a href='https://www.facebook.com/nejc.jelovcan?fref=grp_mmbr_list'><img src="{% static "img/nejcjelovcan.png" %}">Nejc Jelovčan</a></li>
-				<li><a href='https://www.facebook.com/natasa.kristan.35?fref=grp_mmbr_list'><img src="{% static "img/natasamori.png" %}">Nataša Mori</a></li>
-				<li><a href='https://www.facebook.com/barbara.skubic.9?fref=grp_mmbr_list'><img src="{% static "img/barbaraskubic.png" %}">Barbara Skubic</a></li>
-				<li><a href='https://www.facebook.com/maria.cankar?fref=grp_mmbr_list'><img src="{% static "img/marijacankar.png" %}">Marija Cankar</a></li>
-				<li><a href='https://www.facebook.com/tjasa.zajc1'><img src="{% static "img/tjasazajc.png" %}">Tjaša Zajc</a></li>
-				</ul>
-		</div>
-		
+	<ul class="about_list">
+	{% for member in members_list %}
+	{% if member.activity_status == '01' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+	{% endif %}
+	{% if member.activity_status == '02' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Active cat)<li>
+	{% endif %}
+	{% if member.activity_status == '03' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Lazy cat)<li>
+	{% endif %}	
+	{% if member.activity_status == '04' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Sleepy cat)<li>
+	{% endif %}	
+	{% endfor %}
+	</ul>			
+	</div>		
```

Comment:
I think there might be a closing div tag missing... if you indent the content correctly, it might be easier to spot...